src/Migrations/SettingsMigrator.php

```diff
@@ -153,4 +167,24 @@ protected function deleteProperty(string $property): void
 
         $this->repository->deleteProperty($group, $name);
     }
+
+    protected function getCast(string $group, string $name): ?SettingsCast
+    {
+        return optional($this->settingsGroups()->get($group))->getCast($name);
+    }
+
+    protected function settingsGroups(): Collection
+    {
+        return $this
+            ->settingContainer()
+            ->getSettingClasses()
+            ->mapWithKeys(fn(string $settingsClass) => [
+                $settingsClass::group() => new SettingsConfig($settingsClass)
+            ]);
+    }
+
+    protected function settingContainer(): SettingsContainer
+    {
+        return app(SettingsContainer::class);
+    }
```

Comment:
Could you combine these? I don't think we need an extra `settingContainer` methid

Reply:
👍